packages/components/src/components/common/y-axis/as-y-axis.scss

```diff
@@ -1,6 +1,6 @@
 .y-axis {
   --widget-axis-text-color: var(--as--color--type-01, #{$color-type-01});
-  --widget-axis-line-color: rgba(250, 34, 34, 1);
+  --widget-axis-line-color: var(--as--color--ui-05, #{$color-ui-05});
```

Comment:
@elenatorro 

I've switched to --as--color--ui-05 because it's the closest to the older value (type-02). We should not be using  type colors for this so I think it's fine :man_shrugging: